openmrs-19/src/main/resources/webapp/messages/messages.properties

```diff
@@ -2,8 +2,8 @@
 openMRS=OpenMRS
 
 #Lookup names
-openMRS.lookup.motechId=By MotechId
-openMRS.lookup.uuid=By UUID
+openMRS.lookup.motechId=MotechId
+openMRS.lookup.uuid=UUID
```

Comment:
Because there was "Lookup By: By MotchId" and "Lookup By: By UUID" and I thought that twice "By" aint correct.